### ISSUE TITLE
test fragments.md fails with pandoc 2.5.2

### ISSUE THREAD

**Issue by legrostdg** [jaspervdj/patat#70]
pandoc 2.5.2 has just been uploaded to Debian, and patat test fragments.md started failing:

```
Testing tests/fragments.md... files differ
5a6
> 
7a9
> 
14a17
> 
21a25
> 
29a34
> 
```

see https://buildd.debian.org/status/fetch.php?pkg=patat&arch=arm64&ver=0.8.2.5-1&stamp=1567318042&file=log

**Comment by jaspervdj** [comment]
Right, it seems that there are subtle whitespace differences in the Pandoc output in between versions.  I want to keep `patat` fairly simple in this regards because once I start patching Pandoc output, I don't think there will be an end to it.

For now; I think just having the test script ignore whitespace changes will do.

**Comment by jaspervdj** [comment]
@legrostdg This should be fixed in `patat-v0.8.3.0`

**Comment by legrostdg** [comment]
Thanks!
